content/en/docs/Contribution Guidelines/releasing.md

```diff
@@ -3,7 +3,7 @@ title: "Releasing Parquet"
 linkTitle: "Releasing Parquet"
 weight: 4
 description: >
-  How to release Parquet
+  How to release Parquet-Java
```

Comment:
Maybe you can update the `title` to include "Java" as well? (as it's the title that is most visible)

Reply:
done.